integrations/payment_methods/web_pay.md

```diff
@@ -33,7 +33,12 @@ React Native Implementation:
 
 ## Estimation
 
-- Backend: 100hs.
-- Consider extra time for getting production credentials and obtaining Transbank's validation
+### Backend
+
+- The full implementation of this integration was estimated in 100hs. Copying this code should reduce this hours around 30.
```

Comment:
It is reduced to 30 or it is reduced by 30?
Maybe you should use the following wording `... should reduce this estimation to 30hs.`